services/src/main/java/org/keycloak/protocol/oidc/utils/RedirectUtils.java

```diff
@@ -72,8 +72,10 @@ public static Set<String> resolveValidRedirects(KeycloakSession session, String
             if (validRedirect.startsWith("/")) {
                 validRedirect = relativeToAbsoluteURI(session, rootUrl, validRedirect);
                 logger.debugv("replacing relative valid redirect with: {0}", validRedirect);
+                resolveValidRedirects.add(validRedirect);
+            } else {
+                resolveValidRedirects.add(validRedirect);
             }
-            resolveValidRedirects.add(lowerCaseHostname(validRedirect));
```

Comment:
This a nitpick but as it was before (without the `else` block) read better, for me ...

Reply:
:smile: Yep, also for me. I just reverted the previous commit and I didn't re-add that change. It's done now.